exec/java-exec/src/test/java/org/apache/drill/exec/rpc/user/security/TestCustomUserAuthenticator.java

```diff
@@ -56,16 +68,27 @@ public void positiveUserAuth() throws Exception {
     runTest(TEST_USER_2, TEST_USER_2_PASSWORD);
   }
 
-
   @Test
   public void negativeUserAuth() throws Exception {
     negativeAuthHelper(TEST_USER_1, "blah.. blah..");
     negativeAuthHelper(TEST_USER_2, "blah.. blah..");
-    negativeAuthHelper(TEST_USER_2, "");
     negativeAuthHelper("invalidUserName", "blah.. blah..");
   }
 
   @Test
+  public void emptyPassword() throws Exception {
+    try {
+      runTest(TEST_USER_2, "");
+      fail("Expected an exception.");
+    } catch (RpcException e) {
+      final String exMsg = e.getMessage();
+      assertThat(exMsg, containsString("Insufficient credentials"));
```

Comment:
Not part of Kerberos testing, I just followed how the tests were written previously.